src/main/java/org/carlmontrobotics/subsystems/Limelight.java

```diff
@@ -61,21 +78,18 @@ public double getTYDeg(String limelightName) {
   public double getDistanceToSpeakerMeters() {
     if (LimelightHelpers.getFiducialID(SHOOTER_LL_NAME) == RED_SPEAKER_CENTER_TAG_ID
         || LimelightHelpers.getFiducialID(SHOOTER_LL_NAME) == BLUE_SPEAKER_CENTER_TAG_ID) {
-          // TODO: change MOUNT_ANGLE_DEG_SHOOTER
       Rotation2d angleToGoal = Rotation2d.fromDegrees(MOUNT_ANGLE_DEG_SHOOTER)
           .plus(Rotation2d.fromDegrees(getTYDeg(SHOOTER_LL_NAME))); //because limelight is mounted horizontally
       double distance = (SPEAKER_CENTER_HEIGHT_METERS - HEIGHT_FROM_GROUND_METERS_SHOOTER) / angleToGoal.getTan();
       // SmartDashboard.putNumber("limelight distance", distance);
       return distance;
     }
-
     else {
       // SmartDashboard.putNumber("limelight distance", -1);
       return -1;
     }
   }
 
-
   public double getDistanceToNoteMeters() {
```

Comment:
I don't think this is within the context of this PR, but I'm pretty sure this only calculates the distance between the note and camera in the y direction relative to the robot, it doesn't account for the tx measurement from Limelight.

Reply:
Similarly for `getDistanceToSpeakerMeters`. However if it turns out Megatags work just fine, you can probably just get rid of these methods except for the note one (you could make the note method use Megatags)